src/main/java/co/cask/hydrator/plugin/batchSource/KafkaBatchSource.java

```diff
@@ -304,7 +304,10 @@ public Schema getMessageSchema() {
     }
 
     public void validate() {
-      getBrokerMap();
```

Comment:
why not make this nullable? If we ahve a recommendation, then might as well use that as the default value?

Also, there could still be conflicts even if you use the pipeline name as the offset table name, if you have the topic macro-enabled?